tests/test_helpers.py

```diff
@@ -156,6 +162,11 @@
      "20sb_firstprojectum     status:finished\n"
      "Please rerun the helper specifying the complete ID.\n"
      ),
+    (["helper", "v_meetings", "-v"],
+     "grp2020-06-12 does not have a journal club doi\n"
```

Comment:
you will need the logic how check the right exception is raised.  You can google for it, but I think Daniella is already using it (I may have misremembered and it is Adiba)